### PR TITLE
test: fix tests for client-side connections

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

This PR adds tests for #66.

This is mostly the same set of changes repeated over and over again to refactor the existing tests to use the new format. A lot of it is replacing the API response mock data with rippled response mock data.

### Context of Change

Adds tests for #66.

### Type of Change

- [x] Tests (You added tests for code that already exists, or your new feature included in this PR)

### TypeScript/Hooks Update

Not worth the effort to do everything.

- [ ] Updated files to React Hooks
- [ ] Updated files to TypeScript

## Test Plan

CI passes.

### CODE REVIEW

**Review by khancode — APPROVED** [review]
LGTM

File: src/containers/Accounts/AccountHeader/test/actions.test.js [hunk 1]
```diff
@@ -7,6 +7,92 @@ import { initialState } from '../reducer';
...
+  status: 'success',
+  type: 'response',
+};
+
+const moxiosData = {
+  account_info: accountInfo,
+  account_objects: accountObjects,
+  server_info: serverInfo,
+  gateway_balances: gatewayBalances,
+};
```
Comment (ledhed2222) [inline src/containers/Accounts/AccountHeader/test/actions.test.js:76]: put into a fixutre

File: src/containers/Accounts/AccountHeader/test/actions.test.js [hunk 2]
```diff
@@ -20,22 +106,83 @@ describe('AccountHeader Actions', () => {
...
+      balances: { XRP: 123.456 },
+      signerList: undefined,
+      escrows: undefined,
+      paychannels: null,
+      xAddress: {
+        classicAddress: 'rDsbeomae4FXwgQTJp9Rs64Qg9vDiTCdBv',
+        tag: 0,
+        test: false,
+      },
+    };
```
Comment (ledhed2222) [inline src/containers/Accounts/AccountHeader/test/actions.test.js:173]: fixture

File: src/containers/Ledger/test/actions.test.js [hunk 3]
```diff
@@ -73,6 +86,22 @@ describe('Ledger actions', () => {
...
+          command: 'ledger',
+          expand: true,
+          id: 14,
+          ledger_index: LEDGER_INDEX,
+          transactions: true,
+        },
+      },
+      status: 'error',
+      type: 'response',
+    };
```
Comment (ledhed2222) [inline src/containers/Ledger/test/actions.test.js:104]: fixture

File: src/containers/Ledgers/test/LedgersPage.test.js [hunk 4]
```diff
@@ -10,10 +10,33 @@ import { I18nextProvider } from 'react-i18next';
...
+    const postParams = JSON.parse(request.config.data);
+    const { method } = postParams.options;
+    return moxiosData[method];
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  get status() {
+    return 200;
+  }
+}
```
Comment (ledhed2222) [inline src/containers/Ledgers/test/LedgersPage.test.js:35]: general mockresponse that maybe takes a path to a directory that contains all the JSON fixtures

**Review by ledhed2222 — APPROVED** [review]

**Pull request merged**